libs/fgdbg/include/fgdbg/Session.h

```diff
@@ -42,16 +43,24 @@ class Session {
     /**
      * Send session information to server
      */
-    void update() const;
+    void update();
 
 private:
     const std::string& name;
     std::shared_ptr<DebugServer> server;
 
-    // TODO(@raviola) These might need to be a set/map in order to guarantee there are no two [Pass]
-    //  or [Resource] with the same id. This will also allow for updates by key.
-    std::vector<Pass> passes{};
-    std::vector<Resource> resources{};
+    std::unordered_map<size_t, Pass> passes{};
```

Comment:
our code style uses the `m` prefix for private members. e.g. here `mPasses` (and elsewhere)